docs/technical_reference/flowsheets/mvc.rst

```diff
@@ -0,0 +1,125 @@
+Mechanical Vapor Compression
+============================
+
+Introduction
+------------
+
+Mechanical Vapor Compression (MVC) desalination is an evaporative, compressor-driven process where vapor generated
+from saline feedwater is compressed to raise its temperature and pressure. This compressed vapor then condenses,
+releasing latent heat to evaporate the feedwater, thus producing distillate without external heating or cooling.
+The system primarily consists of an evaporator/condenser and a compressor, with optional heat exchangers for enhanced
+heat integration, and may utilize external steam to initiate the process.
+
+Implementation
+--------------
+
+Figure 1 illustrates the process flow diagram for a Mechanical Vapor Compression (MVC) desalination system.
+In this configuration, feedwater is pumped and divided between the cold inlets of a distillate heat
+exchanger (HEX) and a brine HEX. The preheated feed exiting these heat exchangers combines and enters the evaporator,
+where it absorbs heat from the condensing steam and vaporizes, leaving behind concentrated brine. The generated vapor
+is then directed to a compressor, which elevates its temperature and pressure. This compressed vapor is recycled back
+to the evaporator, where it condenses and provides the necessary heat for evaporating the preheated feed. The hot
+concentrated brine and distillate pass through heat exchangers to preheat the incoming feedwater before exiting the
+system. The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only
+   * property package(s) supporting liquid and vapors is provided
+
+.. figure:: ../../_static/flowsheets/mvc.png
+    :width: 600
+    :align: center
+
+    Figure 1. MVC flowsheet
+
+Documentation for each of the WaterTAP unit models can be found below.
+    * `Pressure Changer <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/pressure_changer.html>`_
+    * `Evaporator <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/evaporator.html>`_
+    * `Compressor <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/compressor.html>`_
+    * `Condenser <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/condenser.html>`_
```

Comment:
Since there is no documentation for each component of MVC (compressor, evaporator, condenser), I suggest referring to the MVC unit model instead of linking to each component individually. The current links are non-existent and non-functional. The link to MVC unit model: 
https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html

Reply:
I've updated the compressor, evaporator and condenser links to point to the mvc unit model documentation, and updated the pressure changer to point to the IDAES documentation